client/components/Events/EventDateTime.tsx

```diff
@@ -79,7 +81,7 @@ export class EventDateTime extends React.PureComponent<IProps> {
                     </span>
                 )}
                 <DateTime
-                    withDate={multiDay}
+                    withDate={(showEventStartDate || multiDay)}
```

Comment:
use `??` instead

Reply:
withDate={(showEventStartDate || multiDay)}
its not the same like
withDate={showEventStartDate ?? multiDay}